Tests/SonarQube.TeamBuild.PreProcessor.Tests/RoslynExportProfileTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="RoslynExportProfileTests.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarQube.TeamBuild.PreProcessor.Roslyn;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace SonarQube.TeamBuild.PreProcessor.Tests
+{
+    [TestClass]
+    public class RoslynExportProfileTests
+    {
+        public TestContext TestContext { get; set; }
+
+        #region Tests
+
+        [TestMethod]
+        public void RoslynProfile_LoadValidProfile_Succeeds()
+        {
+            string validXml = @"<?xml version=""1.0"" encoding=""utf-8""?>
+<RoslynExportProfile Version=""1.0"">
```

Comment:
We should consider extending the integration test as well to protect against SQ changing this XML and to make sure we treat template rules correctly.